server/models/Group.ts

```diff
@@ -128,6 +109,9 @@ class Group extends ParanoidModel<
 
   @BelongsToMany(() => User, () => GroupUser)
   users: User[];
+
+  @CounterCache(() => GroupUser, { as: "members", foreignKey: "groupId" })
+  memberCount: Promise<number>;
```

Comment:
All that's needed to create a cached count column 👌

Reply:
Sweet 👌